docs/docs/can-traffic.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Calculating CAN traffic on a bus
+---
+
+
+
+In order to calculate the CAN load, we must declare a few variables and their importance within the CANbus protocol. 
+
+1. Baud: (kbaud)
+
+The baud rate is the rate at which data is transfered in a communication channel (like CAN). For example, 9600 Baud rate is 9600 bits per second
+
+2. Frequency (Hz)
```

Comment:
Frequency should be divided into frequency per message since some message are transmitted more frequently.

For example, I might give you

| message | frequency (Hz) |
| :---|:---|
| BatteryStatus | 10 |
| DebugCode | 1 |
| MotorControl | 500 |

You calculation should calculate the total bits per second from this table. Are all messages likely to have the same number of bits?